test/extended/operators/olm.go

```diff
@@ -0,0 +1,53 @@
+package operators
+
+import (
+	"reflect"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	yaml "gopkg.in/yaml.v2"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Platform][Smoke] OLM should", func() {
```

Comment:
Is the Smoke label intentional? I believe that makes this run before all other tests.

Reply:
I want to follow Clayton's operators test which has Smoke label so they can run together. However, the order in which this test runs doesn't matter. So if you feel the Smoke label is unnecessary, I can remove it.

Reply:
Removed the Smoke label as it seems unnecessary.